sklearn/datasets/svmlight_format.py

```diff
@@ -446,8 +446,8 @@ def dump_svmlight_file(X, y, f,  zero_based=True, comment=None, query_id=None,
 
     Xval = check_array(X, accept_sparse='csr')
     if Xval.shape[0] != yval.shape[0]:
-        raise ValueError("X.shape[0] and y.shape[0] should be the same, got"
-                         " %r and %r instead." % (Xval.shape[0], yval.shape[0]))
+        raise ValueError("X.shape[0] and y.shape[0] should be the same, got "
```

Comment:
please avoid unrelevant changes.

Reply:
Flake 8 change reverted in latest commits.